recipes/metaimageio/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "metaimageio" %}
+{% set version = "1.0.0" %}
+{% set hash = "fa1e0aa1980e10aac3ceb0bfa794b734413f1399c750fd8208dcb2f869ed37df" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - imageio >=2.8
+    - numpy >=1.19
+    - python >=3.6
+
+test:
+  imports:
+    - metaimageio
+
+about:
+  home: https://auneri.github.io/metaimageio
```

I think GitHub URLs are case-senstive:
```suggestion
  home: https://auneri.github.io/MetaImageIO
```
(old one was 404-ing)
```suggestion
  home: https://auneri.github.io/metaimageio
```